yoti-sdk-sandbox/src/main/java/com/yoti/api/client/sandbox/profile/request/attribute/Attribute.java

```diff
@@ -0,0 +1,110 @@
+package com.yoti.api.client.sandbox.profile.request.attribute;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class Attribute {
+    @JsonProperty("name")
+    private String name;
+    @JsonProperty("value")
+    private String value;
+    @JsonProperty("derivation")
+    private String derivation;
+    @JsonProperty("optional")
+    private String optional;
+
+    @JsonCreator
+    private Attribute(@JsonProperty("name") String name,
+            @JsonProperty("value") String value,
+            @JsonProperty("derivation") String derivation,
+            @JsonProperty("optional") String optional) {
+        this.name = name;
+        this.value = value;
+        this.derivation = derivation;
+        this.optional = optional;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public String getValue() {
+        return value;
+    }
+
+    public String getDerivation() {
+        return derivation;
+    }
+
+    public String getOptional() {
+        return optional;
+    }
+
+    public static class AttributeBuilder {
+
+        private String name;
+
+        private String value;
+
+        private String derivation = "";
+
+        private String optional = "false";
+
+        private AttributeBuilder() {
+        }
+
+        public static AttributeBuilder builder() {
+            return new AttributeBuilder();
+        }
+
+        public AttributeBuilder name(String name) {
+            this.name = name;
+            return this;
+        }
+
+        public AttributeBuilder value(String value) {
+            this.value = value;
+            return this;
+        }
+
+        public AttributeBuilder derivation(String derivation) {
+            this.derivation = derivation;
+            return this;
+        }
+
+        public AttributeBuilder optional(String optional) {
+            this.optional = optional;
+            return this;
+        }
+
+        public Attribute build() {
+            return new Attribute(name, value, derivation, optional);
```

Comment:
We should probably assert that _name_ and _attribute_ are not null, if that is our assumption.
Or is that already in place somewhere else?